tests/cephadm/test_cephadm_pull.py

```diff
@@ -0,0 +1,71 @@
+import ast
+
+import requests
+import yaml
+
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def getting_ceph_version(link):
+    """
+    Method return ceph version from magna002 latest ceph container yaml
+    Args:
+        link: magna002 latest ceph container yaml link
+    Returns:
+        ceph version
+    """
+
+    response = requests.get(link)
+    ceph_version = yaml.safe_load(response.text)[5.1]["ceph-version"]
```

Comment:
Let's not hard code the version here, make it generic it should work for all ceph 5.x release.
Get the build version and pass it here.

Reply:
Thanks @sunilangadi2
Removed hard coded value, Add loop to get all the ceph version for ceph 5.x release and verifying same.